homeassistant/components/weather/__init__.py

```diff
@@ -1088,6 +1111,51 @@ async def async_get_forecast_service(
     }
 
 
+async def async_get_forecast_value_service(
+    weather: WeatherEntity, service_call: ServiceCall
+) -> ServiceResponse:
+    """Get weather forecast for specified attribute."""
+    forecast_attribute = service_call.data.get("attribute")
+    forecast_limit_hours = service_call.data.get("limit_hours")
+    forecast_min_max = service_call.data.get("min_max")
```

Comment:
The schema sets a default:
```suggestion
    forecast_min_max = service_call.data["min_max"]
```